assignment.tf

```diff
@@ -0,0 +1,76 @@
+1. Explain the git workflow
+A typical Git workflow involves a series of steps to manage changes in your project’s codebase. Here is an overview of the most common Git workflow used in collaborative projects:
+a. Clone the Repository
+b. Create a New Branch
+c. Make Changes and Commit
+d. Push the Branch to the Remote Repository
+e. Create a Pull Request
+f. Code Review and Feedback
+g. Merge the Pull Request
+f. Delete branch when done and continue cycle when another branch is needed
+
+
+2. How do you clone a git repository
+Inorder to clone a git reop,you got to your repository and copy the url then come to you IDE and use the command git clone 'URL' that will clone your reposiotory.
```

Comment:
no need to add this **Inorder to clone a git reop,you got to your repository and**

Reply:
seen thank you